src/web/components/ParticipantRequests/ApprovedParticipantsTable.stories.tsx

```diff
@@ -18,59 +18,104 @@ export const AllParticipants: Story = {
       {
         id: 1,
         name: 'Participant 1',
-
-        types: [{ id: 1, typeName: 'Type 1' }],
-        status: ParticipantStatus.AwaitingApproval,
-        requestingUser: {
-          email: 'test1@example.com',
-          fullName: 'Test User  1',
-          role: UserRole.Engineering,
-        },
+        types: [],
+        status: ParticipantStatus.Approved,
+        allowSharing: true,
+        completedRecommendations: false,
+        users: [
+          {
+            id: 1,
+            email: 'test1@example.com',
+            firstName: 'First Test User 1',
+            lastName: 'Last Test User 1',
+            role: UserRole.Marketing,
+            acceptedTerms: false,
+          },
+        ],
       },
       {
         id: 2,
         name: 'Participant 2',
+        types: [{ id: 1, typeName: 'Type 1' }],
+        status: ParticipantStatus.AwaitingApproval,
```

Comment:
This component just displays all participants it is given, though we do just plan for it to be for participants that have `ParticipantStatus.Approved`. What do you think about changing the dummy data to reflect that?

Reply:
Yeah I've done that. I thought I already had but must've gotten it mixed up.